### ISSUE TITLE
View filters are exclusive

### ISSUE THREAD

**Issue by fgafford** [geopython/GeoHealthCheck#164]
Views have exclusive filters applied to them. They will only return the last results to be executed. The query should build a single query including ll parameters so that the system to support more then one facet when returning results.  

Code in question: 
https://github.com/geopython/GeoHealthCheck/blob/master/GeoHealthCheck/views.py#L61-L76

**Comment by justb4** [comment]
Oops  commit ca569ca generates error with PostgreSQL backend at 
https://github.com/geopython/GeoHealthCheck/blob/dev/GeoHealthCheck/views.py#L64

```
        filters = filters + ('resource_type = "' + resource_type + '"',)
```

The query contains double quotes for `"OGC:WMS"`: 

```
'SELECT resource.identifier AS resource_identifier, resource.resource_type AS resource_resource_type, resource.active AS resource_active, resource.title AS resource_title, resource.url AS resource_url, resource.latitude AS resource_latitude, resource.longitude AS resource_longitude, resource.owner_identifier AS resource_owner_identifier FROM resource WHERE resource_type = "OGC:WMS"']
```

**Comment by justb4** [comment]
Confirmed: my fix #ced9409 now works with PostgreSQL backend .

**Comment by justb4** [comment]
Ok, merged into master via #169, close and go into maintenance mode. Thanks @fgafford .
